### PR TITLE
Cryptococal menengitis screening variables

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/NDWH/C&T FACT TABLES/load_FactART.sql [hunk 1]
```diff
@@ -390,7 +390,108 @@ select
...
+),
+
+ UnsuppressedAtlastVl as (
+    SELECT
+    PatientPKHash,
+    SiteCode,
+    TestResult,
+    OrderedbyDate
+    from ODS.dbo.Intermediate_OrderedViralLoads
+    where   try_cast (TestResult as decimal) > 1000 and  rank=1
```
Comment (nobert-mumo) [inline Scripts/NDWH/C&T FACT TABLES/load_FactART.sql:402]: @Marymary-dev is it specific here to use Un suppression of   > 1000 copies?

File: Scripts/NDWH/C&T FACT TABLES/load_FactART.sql [hunk 2]
```diff
@@ -390,7 +390,108 @@ select
...
+ConfirmedTreatmentFailure as (
+    SELECT
+    UnsuppressedAtlastVl.PatientPKHash,
+    UnsuppressedAtlastVl.SiteCode,
+    UnsuppressedAtlastVl.TestResult,
+    UnsuppressedAtlastVl.OrderedbyDate
+   from  UnsuppressedAtlastVl
+   inner join UnsuppressedAtsecondLastVL on UnsuppressedAtlastVl.PatientPKHash=UnsuppressedAtsecondLastVL.PatientPKHash and UnsuppressedAtlastVl.SiteCode=UnsuppressedAtsecondLastVL.SiteCode
+
+),
```
Comment (nobert-mumo) [inline Scripts/NDWH/C&T FACT TABLES/load_FactART.sql:423]: @Marymary-dev I think for the longitudinal datasets we have to partition or something to make sure we don't introduce dups

**Review by nobert-mumo — CHANGES_REQUESTED** [review]

**Review by DennisGibz — COMMENTED** [review]
I have tested the scripts and its taking too long to run. 2hrs and it hasn't finished.

**Review by DennisGibz — APPROVED** [review]
I have tested the scripts and its taking too long to run. 2hrs and it hasn't finished.

**Comment by Marymary-dev** [comment]
@DennisGibz  it is taking approximately 47 minutes to run

**Review by DennisGibz — APPROVED** [review]

**Review by cbrianbet — APPROVED** [review]

File: Scripts/NDWH/C&T FACT TABLES/load_FactART.sql (hunk 2 above)
Comment (nobert-mumo) [reply Scripts/NDWH/C&T FACT TABLES/load_FactART.sql:423]: @Marymary-dev would this be viable ?

**Review by nobert-mumo — APPROVED** [review]

**Pull request merged**